t/03-testapi.t

```diff
@@ -1286,6 +1285,28 @@ subtest 'upload_asset and parse_junit_log' => sub {
     like(exception { parse_junit_log('foo') }, qr/foo/, 'parse_junit_log calls upload_logs');
 };
 
+subtest 'ensure_installed test' => sub {
+    like(exception { ensure_installed }, qr/implement.*for your distri/, 'ensure_installed can be called');
+
+    my $mock_testapi = Test::MockModule->new('testapi');
+    $mock_testapi->redefine(
+        x11_start_program => sub {
+            my ($program, @args) = @_;
+            return ($program, @args);
+        }
+    );
+    $mock_testapi->redefine(
+        wait_still_screen => sub {
+            return => 1;
+        }
+    );
```

Comment:
Seems like you are not doing anyting with the mocked functions so probably just a "noop" will be enough

```suggestion
    $mock_testapi->noop('x11_start_program', 'wait_still_screen');
```